python/lsst/drp/tasks/assemble_cell_coadd.py

```diff
@@ -359,7 +403,7 @@ def run(self, inputWarps, skyInfo, **kwargs):
                 outer=image_planes,
                 psf=cell_coadd_psf.computeKernelImage(cell_coadd_psf.getAveragePosition()),
                 inner_bbox=cellInfo.inner_bbox,
-                inputs=None,  # TODO
+                inputs=frozenset(observation_identifiers_gc[cellInfo.index]),
```

Comment:
If we use a `frozenset` or `set` here we need to make sure we sort on save to avoid making the persisted order dependent on salted hashing.  Might be better to remove duplicates then sort it and make a `tuple` right here.

Reply:
Also, we really don't want to be using `getAveragePosition` two lines above.  That is the average position of all the stars that went into the PSF model, and it's not necessarily going to be anywhere near this cell.

Reply:
The PSF coaddition, including dropping the use of `getAveragePosition`, is coming next in DM-43515.

Reply:
Do we care about the ordering here when persisting. I can see if we try to compare if two files are identical without even opening them, but is that something we expect to support?

Reply:
I'm not sure we can make it all the way to full support, but I think it's good practice to minimize unnecessary differences.

Reply:
Alright, in the [other PR](https://github.com/lsst/cell_coadds/pull/31), I added a `__lt__` operator for `ObservationIdentifiers` and sort the inputs in the `__init__` method of `SingleCellCoadd`.